packages/client/manifest.json

```diff
@@ -3157,6 +3157,57 @@
       }
     ]
   },
+  "codescanner": {
+    "name": "Barcode/QR Scanner",
+    "icon": "Camera",
+    "styles": [
+      "size"
+    ],
+    "draggable": true,
```

Comment:
NAB: setting draggable to true here is redundant as it's only a flag used to disable DND if set to false. Setting illegal children is also redundant since this component doesn't take child components anyway.